<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_create">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>158</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Создать новую задачу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_change">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>158</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменить новую задачу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>158</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удаление задачи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>158</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вывод задач</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_else">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>158</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сколько жить осталось?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_settings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>158</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="area_name">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="area_start">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="area_end">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_create">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Создание задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_change">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Изменение задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_delete">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Удаление задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_output">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вывод задачи</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_settings">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_complete">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Список дел изменен!</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_date_err">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Неверная дата!</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_name_err">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имя не должно быть пустым!</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="area_task_list">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="area_task_num">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_take">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выб.</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_num_err">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Неверный номер задачи!</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_copy_err">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Такая задача уже есть!</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_gante">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Диаграмма Ганта</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_list">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Список дел</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_type">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Как отобразить задачи?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_type_get">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Показать</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_type_err">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Пункт не выбран!</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_life">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вам осталось жить ... дней.</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_else_days">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>До смерти осталось ... дней.</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_burn_days">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ты прожил ... дней.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="area_burthdate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="area_deathdate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>190</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="text_death">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата смерти</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_burth">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата рождения</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_theme">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Тема:</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_settings_complete">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Настройка сохранена!</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_dark">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Тёмная</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radio_light">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Светлая</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_delete_everything">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить все задачи</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>503</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
